server/src/test/python/cromwell-az-e2e-test/cromwell_az_e2e_test/az_e2e.py

```diff
@@ -0,0 +1,230 @@
+                   "accept": "application/json"}
+        # First call to initiate workspace deletion
+        response = requests.delete(url=delete_workspace_url, headers=headers)
+        output_message(response.text)                              
+        handle_failed_request(response, f"Error deleting workspace {workspace_name} - {workspace_namespace}", 202)
+        output_message(f"Workspace {workspace_name} - {workspace_namespace} delete request submitted")
+       
+        # polling to ensure that workspace is deleted (which takes about 5ish minutes)
+        is_workspace_deleted = False
+        token_expired = False
+        while not is_workspace_deleted and not token_expired:
+            time.sleep(2 * 60)
+            get_workspace_url = f"{rawls_url}/api/workspaces/{workspace_namespace}/{workspace_name}"
+            polling_response = requests.get(url=get_workspace_url, headers=headers)
+            polling_status_code = polling_response.status_code
+            output_message(f"Polling GET WORKSPACE - {polling_status_code}")
+            if polling_status_code == 200:
+                output_message(f"Workspace {workspace_name} - {workspace_namespace} is still active")
+            elif polling_status_code == 404:
+                is_workspace_deleted = True
+                output_message(f"Workspace {workspace_name} - {workspace_namespace} is deleted")
+            elif polling_status_code == 401:
+                token_expired = True
+            else:
+                output_message(f"Unexpected status code {polling_status_code} received\n{polling_response.text}")
+                raise Exception(polling_response.text)
+        if token_expired:
+            raise Exception(f"Workspace {workspace_name} was not deleted within bearer token lifespan")
+        
+def test_cleanup(workspace_namespace, workspace_name):
+    try:
+        delete_workspace(workspace_namespace, workspace_name)
+        output_message("Workspace cleanup complete")
+    # Catch the exeception and continue with the test since we don't want cleanup to affect the test results
+    # We can assume that Janitor will clean up the workspace if the test fails
+    except Exception as e:
+        output_message("Error cleaning up workspace, test script will continue")
+        output_message(f'Exception details below:\n{e}')
+
+def start():
+    workspace_namespace = ""
+    workspace_name = ""
+    workspace_id = ""
+    found_exception = False
+    
+    # Sleep timers for various steps in the test
+    workflow_run_sleep_timer = 60 * 5
+    provision_sleep_timer = 60 * 15
+    workflow_status_sleep_timer = 60 * 2
+    
+    try:
+        created_workspace = create_workspace()
+        workspace_id = created_workspace['workspaceId']
+        workspace_namespace = created_workspace['namespace']
+        workspace_name = created_workspace['name']
+        time.sleep(provision_sleep_timer) # Added an sleep here to give the workspace time to provision
```

Comment:
Not a blocker for this PR, but it might be nice for this to poll rather than sleep in order to save a couple minutes.